src/site/content/en/fast/lazy-loading/index.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Lazy-loading
+authors:
+  - jeremywagner
+  - rachelandrew
+date: 2019-08-16
+updated: 2020-06-09
+description: |
+  This post explains lazy-loading and why you might want to lazy-load elements on your site.
+tags:
+  - performance
+---
+
+- [What is lazy-loading](#what)
+- [Why lazy-load images or video?](#why)
+- [Implementing lazy-loading](#implementing)
```

Comment:
```suggestion
```

Reply:
Nitpick: Since this is a fairly short read I feel the TOC is not really necessary. There's no standard TOC component (yet) and the DIY version feels a bit tacked on.

Reply:
I agree, it was leftover from when it was all one article, probably don't need it now.